includes/cf/radarr-optional-uhd.md

```diff
@@ -7,4 +7,4 @@
 
     Breakdown and Why
 
-    - **{{ radarr['cf']['sdr']['name'] }}:** This will help to prevent to grab UHD/4k releases without HDR Formats.
+    - **{{ radarr['cf']['sdr']['name'] }}:** This will prevent grabbing UHD/4k releases without HDR Formats.
```

Comment:
```suggestion
    - **{{ radarr['cf']['sdr']['name'] }}:** This will prevent grabbing UHD/4K releases without HDR Formats.
```

Reply:
Going to decline this one, and use the same copy you used in the Sonarr file.